frontend/src/tests/api.test.js

```diff
@@ -76,4 +76,7 @@ describe("Mock API tests", () => {
     describe.skip("`/instructors` tests", () => {
         instructorsTests(mockAPI);
     });
+    describe("`/applications` tests", () => {
+        applicationsTests(mockAPI);
```

Comment:
the frontend files shouldn't be changed, I suspect that you created this branch out of your applicant tests branch instead of master, so it has the previous commit.